### PR TITLE
LPS-134556 Cover WorkflowLogResource with integration tests

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:sf

File: modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowLogResourceTest.java [hunk 1]
```diff
@@ -15,14 +15,401 @@
...
+			PermissionCheckerFactoryUtil.create(
+				UserLocalServiceUtil.getUser(TestPropsValues.getUserId())));
+	}
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		BaseWorkflowLogResourceTestCase.setUpClass();
```
Comment (MarinhoFeliphe) [inline modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowLogResourceTest.java:87]: Hi, @seltong 

Could you remove this `BaseWorkflowLogResourceTestCase.setUpClass();`, please?

thanks!

File: modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowLogResourceTest.java [hunk 2]
```diff
@@ -15,14 +15,401 @@
...
+
+		_siteContentReviewerRole = _roleLocalService.getRole(
+			testGroup.getCompanyId(), RoleConstants.SITE_CONTENT_REVIEWER);
```
Comment (MarinhoFeliphe) [inline modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowLogResourceTest.java:90]: We can move this `_siteContentReviewerRole = ...` to setUpClass method?

File: modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowLogResourceTest.java [hunk 3]
```diff
@@ -15,14 +15,401 @@
...
+
+	@Override
+	@Test
+	public void testGetWorkflowInstanceWorkflowLogsPage() throws Exception {
+		WorkflowTask workflowTask = WorkflowTaskTestUtil.getWorkflowTask(
+			_workflowInstance.getId());
+
+		User user = TestPropsValues.getUser();
+
+		Assignee assignee = AssigneeTestUtil.addAssignee(user.getGroup());
```
Comment (MarinhoFeliphe) [inline modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowLogResourceTest.java:105]: The `AssigneeTestUtil.addAssignee` method creates an user, we need create this user or we can get this other one`TestPropsValues.getUser()`?

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

File: modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowLogResourceTest.java [hunk 4]
```diff
@@ -15,14 +15,388 @@
...
+						description = _language.format(
+							_getResourceBundle(),
+							"x-assigned-the-task-to-himself",
+							new Object[] {
+								_portal.getUserName(
+									TestPropsValues.getUserId(),
+									StringPool.BLANK),
+								_portal.getUserName(
+									TestPropsValues.getUserId(),
+									StringPool.BLANK)
```
Comment (MarinhoFeliphe) [inline modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowLogResourceTest.java:130]: Hi, @seltong

for this property `x-assigned-the-task-to-himself` we need only one parameter.

File: modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowLogResourceTest.java [hunk 5]
```diff
@@ -15,14 +15,388 @@
...
+						workflowTaskId = workflowTask.getId();
+					}
+				}),
+			workflowLogs2);
+	}
+
+	@Override
+	protected String[] getAdditionalAssertFieldNames() {
+		return new String[] {
+			"commentLog", "description", "state", "workflowTaskId"
```
Comment (MarinhoFeliphe) [inline modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowLogResourceTest.java:355]: Hi, @seltong

I guess the `"type"` an interesting field to compare in equals method, what do you think?

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Review by MarinhoFeliphe — APPROVED** [review]

**Comment by MarinhoFeliphe** [comment]
ci:forward

**Comment by MarinhoFeliphe** [comment]
Just started reviewing :)

**Pull request closed without merging**